api/src/DataFixtures/AppFixtures.php

```diff
@@ -10,34 +10,64 @@
 
 class AppFixtures extends Fixture
 {
+    /** @var \Doctrine\Persistence\ObjectManager $objectManager */
```

Comment:
Please can you add `\Doctrine\Persistence\ObjectManager` to the imports and change the docblock to `ObjectManager`